faqrenderer/faq.md

```diff
@@ -0,0 +1,121 @@
+
+1. In the top left corner, open a terminal by going to File->New->Terminal or open a New Launcher (Ctrl+Shift+L) and select terminal from the 'other' section.
+
+  ![Finding the Terminal](terminal.png)
+
+2. Within the terminal, use the command `conda create -n 'your_env_name' 'kernel' 'packages'` to create your environment.
+Below is an example that creates an environment called `scipkg`, which contains the `ipykernel` kernel along with the python packages `numpy`, `pandas` and `matplotlib`. In general, you may include any packages which are available through conda.
+
+  ![Conda Create command](conda-create.png)
+
+  *Do not worry if you recieve a warning about newer versions of conda when running the `conda create` command. Just proceed through the installation when prompted by pressing `y` or by initially running `conda create` with a `-y` flag at the end.*
+
+3. After the environment has been built, activate it using `conda activate your_env_name`. You should now see `(your_env_name)` at the beginning of the terminal prompt. This indicates that the environment is active. Note that the environment will only be active within that specific terminal session. To deactivate, run `conda deactivate`.
+
+  ![Conda Activate command](conda-activate.png)
+
+4. If you specified a Jupyter kernel then you may run your environment as a Jupyter notebook or console. Open the launcher (Ctrl+Shift+L) and select your environment from there. Using the `scipkg` example, the environment will appear as `Python [conda env:.conda-scipkg]`. You may have to wait a little bit for the icon to appear; refresh the webpage if needed.
+
+  ![Launch the conda environment](env-launcher.png)
+
+  Once the notebook opens, you can verify that you are using the proper environment by looking in the top right corner;
+
+  ![Notebook environment](notebook.png) 
+
+5. Now your conda environment is created! You may access it in the terminal by `conda activate 'your_env_name'` or as a notebook from the launcher. 
+
+Note that if you want to use your environment with a Jupyter notebook, you must install a [Jupyter kernel](#what-is-a-jupyter-kernel) along with your packages. Here's a list of common languages and their kernels which are supported through conda:
+
+| Language | Jupyter kernel |
+| - | - |
+| Python | ipykernel |
+| R | r-irkernel |
+| C++ | xeus-cling |
+| Octave | octave_kernel |
+
+You may view all your created conda environments using `conda env list`. To remove an environment, use `conda env remove -n your_env_name`. For reference, all of your created environments are stored in `/home/jovyan/.conda/envs/`. You can learn more about [managing environments](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) and [more conda commands](https://docs.conda.io/projects/conda/en/latest/commands.html) from the linked documentation.
+
```

Comment:
This conda env creation is really nicely done. Much clearer than before!

Reply:
You should probably warn them about disk space limits. If we are only giving like 2Gb per user, you can kill that with a single conda env pretty quickly. We should probably give people 5 to 10gb at least.

Reply:
We can change it to 10GB or potentially give them even more storage than that. 10GB is probably a good starting point though. Let me know and we'll mention it in the FAQ and change that in the helm values.

Reply:
Let's give everyone 10 GB.